### DISCUSSION TITLE
Discuss adding CIS's unofficial ESX schema to OVAL 6.0 (replacing existing/obsolete ESX schemas?)

### DISCUSSION THREAD

**Discussion by vanderpol** [OVAL-Community/OVAL#207] [General]
When we were discussing the usefulness of OVAL tests back in #165 @maxullman documented the following tests, which appear to have come from CIS?

cmd shellcommand_test
esx host_acceptancelevel_test
esx host_account_test
esx host_advancedsetting_test
esx host_authentication_test
esx host_busadapter_test
esx host_coredump_test
esx host_firewallexception_test
esx host_lockdown_test
esx host_module_test
esx host_ntpserver_test
esx host_portgroup_test
esx host_service_test
esx host_vib_test
esx host_vswitchpolicy_test
esx host_webserverssl_test
esx vds_portgroup_test
esx vds_test
esx version_test
esx vm_advancedsetting_test
esx vm_device_test
esx vm_harddiskdevice_test

The schema reside at https://github.com/joval/jOVAL/blob/master/scap/schemas/oval/unofficial/esx-definitions-schema.xsd

As I'm currently removing the existing ESX schema as they have never been used, it seems like a good time to essentially replace them with ones that we know have been used.

@solind or @A-Biggs any thoughts on this?

**Comment by solind** [comment]
I think the Arctic Wolf version is the most usable version of the ESX schema, so yes, I agree with you @vanderpol .

**Reply by vanderpol** [reply]
I'll plant to replace the existing ESX schema with the Arctic Wolf versions.
